opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -50,6 +52,9 @@ pub trait LogProcessor: Send + Sync + Debug {
     #[cfg(feature = "logs_level_enabled")]
     /// Check if logging is enabled
     fn event_enabled(&self, level: Severity, target: &str, name: &str) -> bool;
+
+    /// Set the resource for the log processor.
+    fn set_resource(&self, resource: &Resource);
```

Comment:
I suggest to provide an empty implementation here. Not all processor author will be interested in doing anything with Resource, so its best to make it optional.

Reply:
Good suggestion. Done.